atomicfu-gradle-plugin/build.gradle

```diff
@@ -98,3 +98,7 @@ if (isKotlinVersionAtLeast(ext.kotlin_version, 1, 9)) {
         exclude "**/JsLegacyTransformationTest*", "**/MppLegacyTransformationTest*"
     }
 }
+
+kotlin {
```

Comment:
Replacing 
```
sourceCompatibility = JavaVersion.VERSION_1_8
targetCompatibility = JavaVersion.VERSION_1_8
```

with 
```
toolchain {
    languageVersion.set(JavaLanguageVersion.of(8))
}
```
in the root build.gradle file should solve the issue without specifying `jvmToolchain` for each individual plugin.

Reply:
fixed